python/sglang/srt/layers/amx_utils.py

```diff
@@ -2,10 +2,13 @@
 
 import torch
 
-from sglang.srt.utils import cpu_has_amx_support
+from sglang.srt.utils import cpu_has_amx_support, get_bool_env_var
 
 logger = logging.getLogger(__name__)
 
+import os
+
+SGLANG_USE_CPU_INT4_W4A8 = get_bool_env_var("SGLANG_USE_CPU_INT4_W4A8")
```

Comment:
why we need an env to tell us to kick off int4 w4a8?

Reply:
This PR (and kernels in https://github.com/sgl-project/sglang/pull/8226) contains w4a8 GEMM and  w4a16 GEMM/MoE